lib/ansible/modules/cloud/amazon/elasticache_subnet_group.py

```diff
@@ -58,15 +58,14 @@
 '''
 
 try:
-    import boto
-    from boto.elasticache import connect_to_region
-    from boto.exception import BotoServerError
-    HAS_BOTO = True
+    import boto3
+    import botocore
+    HAS_BOTO3 = True
 except ImportError:
-    HAS_BOTO = False
+    HAS_BOTO3 = False
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.ec2 import HAS_BOTO, ec2_argument_spec, get_aws_connection_info
+from ansible.module_utils.ec2 import HAS_BOTO3, boto3_conn, ec2_argument_spec, get_aws_connection_info
```

Comment:
With AnsibleAWSModule you can remove all of these imports.
Creating the connection will just be `connection = module.client('elasticache')`

Reply:
@s-hertel Updated with required changes.